content/kapp/docs/latest/config.md

```diff
@@ -71,7 +71,8 @@ diffMaskRules:
 - `rebaseRules` (array) list of rebase rules
   - `path` (array) specifies location within a resource to rebase. Mutually exclusive with `paths`. Example: `[spec, clusterIP]`
   - `paths` (array of paths) specifies multiple locations within a resource to rebase. This is a convenience for specifying multiple rebase rules with only different paths. Mutually exclusive with `path`. Available in v0.27.0+.
-  - `sources` (array of strings) specifies source preference from where to copy value from. Allowed values: `new` or `existing`. Example: `[new, existing]`
+  - `type` (string) specifies strategy to modify field values. Allowed values: `copy` or `remove`. `copy` will transfer and update the field value; `remove` will delete the field. 
+  - `sources` (array of strings) specifies source preference from where to copy value from. Allowed values: `new` or `existing`. `new` and `existing` refer to resources, `new` being a resource from user input, where `existing` is an existing resource in the cluster. Examples: `[new, existing]`, `[existing]`, `[existing, new]`
```

Comment:
Having read this and our current docs half a dozen times, I still don't understand this sources field. I'd love to talk with someone and play with the wording.

_I think_ it refers to whether kapp should use the existing value on the cluster, or overwrite it, but I'm very unsure.